src/main/java/com/icthh/xm/uaa/domain/GrantType.java

```diff
@@ -10,7 +10,7 @@
 @AllArgsConstructor
 public enum GrantType {
 
-    OTP("otp"),
+    OTP("otp_token"),
```

Comment:
in this case we have no token, just otp, and in case of grand type code we have code or grand type password we have password